package.json

```diff
@@ -414,6 +414,10 @@
                                         "cljsDefaultBuild": {
                                             "type": "string",
                                             "description": "Which cljs build to attach to at the initial connect."
+                                        },
+                                        "projectRootDir": {
+                                            "type": "string",
+                                            "descripion": "Defines the project root directory.  Useful for working with monorepos."
```

Comment:
I think it needs to be clear that there is a project root dir dynamically defined even without this setting. As it is worded now, for someone just reading the first sentence (yes, those people exist 😄 ) it can leave the impression that this setting is needed. Consider pasting the text you wrote for the user docs. It makes the purpose of the setting crystal clear, imo.

Reply:
Good call, fixed.